src/v3/src/components/InputText/InputText.tsx

```diff
@@ -31,130 +25,58 @@ import {
   UISchemaElementComponent, UISchemaElementComponentWithValidationProps,
 } from '../../types';
 import { getTranslation } from '../../util';
-import FieldLevelMessageContainer from '../FieldLevelMessageContainer';
 import { withFormValidationState } from '../hocs';
 
 const InputText: UISchemaElementComponent<UISchemaElementComponentWithValidationProps> = ({
-  type,
   uischema,
   errors,
   handleChange,
   handleBlur,
-  describedByIds,
 }) => {
   const value = useValue(uischema);
   const { loading } = useWidgetContext();
   const {
     translations = [],
     focus,
     parserOptions,
-    showAsterisk,
-    dir,
   } = uischema;
   const label = getTranslation(translations, 'label');
   const hint = getTranslation(translations, 'hint');
   const explain = getTranslation(translations, 'bottomExplain');
-  const optionalLabel = getTranslation(translations, 'optionalLabel');
   const {
     attributes,
     inputMeta: { name, required },
     dataSe,
   } = uischema.options;
+  const { autocomplete, inputmode } = attributes || {};
```

Comment:
The only values in `attributes` are `autocomplete` and `inputmode` so we are not losing any attributes by not spreading the object as props passed to `InputBase` like the old code was doing